src/pages/2022.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from "react"
```

Comment:
Hey @zastrow is there a reason for wanting to use Class components? Seems like the completely of Page2022 is simple enough to easily convert to a functional component.

Reply:
Honestly, I copied over from previous years. But I updated the 2022 file.